Scripts/Runtime/Entities/TaskSystem/Job/JobConfig/UpdateJobConfig.cs

```diff
@@ -42,7 +40,7 @@ private void RequireRequestCancelDataStreamForRead(CancelRequestsDataStream canc
                              new CancelRequestsAccessWrapper(cancelRequestsDataStream, AccessType.SharedRead, byte.MaxValue));
         }
         
-        private void RequireCancellableTaskStreamForWrite(CancellableTaskStream<TInstance> cancellableTaskStream)
+        private void RequireCancellableTaskStreamForWrite(TaskStream<TInstance> cancellableTaskStream)
         {
```

Comment:
Do you now need to assert that `cancellableTaskStream.IsCancellable == true`?

Reply:
I had thought no because it's a private function and is gated by an if check above. 

But sure, I added the check just in case code changes happen.